moda-contracts/test/SplitsFactoryWithFork.t.sol

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.21;
+
+import {SplitsFactory} from "../src/SplitsFactory.sol";
+import {ISplitsFactory} from "../src/interfaces/ISplitsFactory.sol";
+import {IManagement} from "../src/interfaces/IManagement.sol";
+import {ISplitMain} from "../src/interfaces/0xSplits/ISplitMain.sol";
+import {IOfficialModaContracts} from "../src/interfaces/IOfficialModaContracts.sol";
+import {SplitMainMock} from "./mocks/SplitMainMock.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "forge-std/Test.sol";
+import "forge-std/console2.sol";
+
+contract ModaRegistry is IOfficialModaContracts {
+    address _treasury;
+    uint32 _fee;
+    uint32 _feeMultiplier;
+
+    constructor(address treasury, uint32 fee, uint32 feeMultiplier) {
+        _treasury = treasury;
+        _fee = fee;
+        _feeMultiplier = feeMultiplier;
+    }
+
+    function getTreasuryInfo() external view returns (address, uint32, uint32) {
+        return (_treasury, _fee, _feeMultiplier);
+    }
+
+    function getSplitsFactory() external pure returns (ISplitsFactory) {
+        return ISplitsFactory(address(0));
+    }
+
+    function getManagement() external pure returns (IManagement) {
+        return IManagement(address(0));
+    }
+}
+
+contract SplitsFactoryWithForkTest is Test {
+    SplitsFactory _splitsFactory;
+    ModaRegistry _modaRegistry;
+    ISplitMain _splitMain;
+
+    string POLYGON_RPC_URL = vm.envString("POLYGON_RPC_URL");
```

Comment:
Can we change this to `RPC_URL`? What if we want to fork Mainnet? env vars would be confusing